samples/scenarios/snowflake_oauth/README.md

```diff
@@ -0,0 +1,149 @@
+
+# Readme
+
+This sample connector plugin demonstrates how to add OAuth Authentication type to Snowflake datasource.  This sample plugin code has been tested for all the authentication methods listed here with all platforms (Mac Desktop, Windows Desktop and Server, Linux Server). You can use this as an example for how to add OAuth to your own connector. 
+For Under Development connetor, refer to this page on how to enable it locally: https://tableau.github.io/connector-plugin-sdk/docs/share.
+
+## <a id="purpose"/> Use case(s) of this connector plugin
+1. Use **OAuth** authentication to connect to Snowflake with `Tableau Desktop on Windows`/`Tableau Desktop on Mac`. See [OAuth for Tableau Desktop](#desktop)    
+1. Publish a viz from `Tableau Desktop` using **Prompt** and open it in Tableau Server. See [OAuth for Tableau Server](#server)
+1. Publish a viz from `Tableau Desktop` using **Embed [your_username]** and open it in Tableau Server. See [OAuth for Tableau Server](#server)
```

Comment:
This section is specific to this connector sample, but the rest seems more general. Willing to be pushed back on this, but can we replace it with a link to the documentation so we don't have to worry about keeping this readme up-to-date with future changes?